upstream/solr6/local/92-init-solr.sh

```diff
@@ -275,6 +275,11 @@ then
     setJavaOption 'ssl-truststore.aliases' "-Dssl-truststore.aliases=ssl-alfresco-ca,ssl-repo-client"
     setJavaOption 'ssl-truststore.ssl-alfresco-ca.password' "-Dssl-truststore.ssl-alfresco-ca.password=$SSL_TRUST_STORE_PASSWORD"
     setJavaOption 'ssl-truststore.ssl-repo-client.password' "-Dssl-truststore.ssl-repo-client.password=$SSL_TRUST_STORE_PASSWORD"
+else
+    SECURITY_FILE=${SOLR_INSTALL_HOME}/solrhome/security.json
+    if [ -f "$SECURITY_FILE" ]; then
+        rm -f $SECURITY_FILE
```

Comment:
The security file looks as follows:

`/opt/alfresco-search-services/solrhome# cat security.json 
{
  "authentication" : {
    "class": "org.alfresco.solr.security.SecretSharedAuthPlugin"
  }
}`

Ideally the file should remain and the "authentication" key and it's values should be removed.

I tried doing this with a sed command but couldn't get it working...

Reply:
jq is probably a better tool to delete something from json